File: app/src/main/java/com/example/cb/test/mvp/presenter/view/INewListView.java
package com.example.cb.test.mvp.presenter.view;

import com.example.cb.test.rx.http.NewsResp;

/**
 * Created by bin on 2018/12/24.
 */
public interface INewListView {
    void onRequestSuccess(NewsResp data);

    void onError(String error);
}

File: app/src/main/java/com/example/cb/test/mvp/view/INewListView.kt
package com.example.cb.test.mvp.view

import com.example.cb.test.rx.http.NewsResp

/**
 * Created by bin on 2018/12/24.
 */
interface INewListView {
    fun onRequestSuccess(data: NewsResp)

    fun onError(error: String)
}
